python/foglamp/common/service.py

```diff
@@ -0,0 +1,51 @@
+from enum import IntEnum
```

Comment:
This code is used by the storage_client to validate if an entry coming back from core is a valid service entry. It used to be in https://github.com/foglamp/FogLAMP/blob/develop/python/foglamp/services/common/microservice_management/service_registry/instance.py